_security-plugin/configuration/yaml.md

```diff
@@ -130,6 +131,12 @@ The opensearch.yml file also contains the `plugins.security.allow_default_init_s
 plugins.security.allow_default_init_securityindex: true
 ```
 
+Authentication cache for the security plugin exists to help speed up authentication by temporarily storing user credentials, thereby bypassing the authentication backend and the response times involved with that process. You can set a timeout value for cashing in minutes with the `plugins.security.cache.ttl_minutes` property. Default is `60`. You can disable caching by adding the value `0`.
```

Comment:
"_caching_ in minutes"

"...temporarily storing user credentials" - What is cached is not the credentials, rather the user object that is returned from the authentication backend to prevent the security plugin from having to make repeated requests to the authentication backend which can be lengthy. 

Instead of `adding the value 0`, what about `setting the value to 0`?

Reply:
@cwperks Thanks for the review. Reworded and fixed typo based on the comments. Appreciate it.